src/activities/solar_system/Dataset.js

```diff
@@ -0,0 +1,349 @@
+/* GCompris - Dataset.js
+ *
+ * Copyright (C) 2017 Aman Kumar Gupta <gupta2140@gmail.com>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+function get() {
+    return [
+                {
+                    "clipImg": "qrc:/gcompris/src/activities/solar_system/resource/sun_clip.svg",
+                    "realImg": "qrc:/gcompris/src/activities/solar_system/resource/sun_real.svg",
+                    "bodyName": qsTr("Sun"),
+                    "levels": [
+                        { 	//sub-level 1
+                            "question": qsTr("The sun is the_____ largest object in our solar system"),
+                            "options": [qsTr("Sixth"),qsTr("Third"),qsTr("First"),qsTr("Seventh")],
+                            "closeness": ["17.5%","67%","100%","1%"]
+                        },
+                        { 	//sub-level 2
+                            "question": qsTr("The temperature of the sun is around _____degrees"),
+                            "options": ["1000","4500","5505","3638"],
+                            "closeness": ["1%","78%","100%","60%"]
+                        },
+                        { 	//sub-level 3
+                            "question": qsTr("Sun is ___ years old"),
+                            "options": [qsTr("1.2 billion"),qsTr("3 billion"),qsTr("7 billion"),qsTr("4.5 billion")],
+                            "closeness": ["1%","55%","25%","100%"]
```

Comment:
do we really want to display 0.1% differences?

Reply:
umm....i'm not quite sure of it. But 0.1% is too small value when represented in the closeness meter, so I think in that point of view, we may ignore it.